parm/config/config.init

```diff
@@ -13,9 +13,11 @@ echo "BEGIN: config.init"
 
 export UFS_DIR=${HOMEgfs}/sorc/ufs_utils.fd
 export GDASINIT_DIR=${UFS_DIR}/util/gdas_init
+export GDAS_INIT_DIR=${UFS_DIR}/util/gdas_init
```

Comment:
Why are you introducing a second variable that does the exact same thing? I prefer this new form, but the old one would need to be removed and any existing references updated.

Reply:
```suggestion
```

Reply:
We can commit this suggestion and merge the PR.
If this breaks, then someone will need to be assigned to fix it.